### DISCUSSION TITLE
Any way to check if the usage is supported on native platforms?

### DISCUSSION THREAD

**Discussion by loggerhead** [nativewind/nativewind#930] [Ideas]
**Description**
I'm using NativeWind to build iOS and Android apps and have noticed significant differences between web and native platforms. I want to identify all the incompatible usages so that I can address them.

**Expected behavior**
A tool or plugin to identify all "Web only" usages in the project.

**Screenshots**
<img width="676" alt="image" src="https://github.com/marklawlor/nativewind/assets/5056390/b8031c2f-a429-420d-9a6b-6d22b5210b2c">

**Comment by feledori** [comment]
This is a oss project build the tool or plugin and contribute.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
